pl/django_orm/README.md

```diff
@@ -202,7 +202,7 @@ Możemy także odwrócić kolejność poprzez dodanie `-` na początku:
 
 ### Łączenie QuerySetów
 
-Możesz też połączyć QuerySety:
+Jak widziałyśmy, niektóre metody `Post.objects` zwracają nam QuerySet. Te same metody z kolei również może być wywoływana na QuerySet, a następnie zwróci nam nowy QuerySet. W związku z tym, można połączyć ich wpływ przez **złączenie** ich razem:
```

Comment:
"Te same metody z kolei również może być wywoływana na QuerySet" this part is confusing. Could we change that? "Te same metody można wywoływać na QuerySet" hehh, this is hard. :C 

"W związku z tym, można połączyć ich wpływ przez **złączenie** ich razem:" ---> W związku z tym, możemy połączyć dwie lub więcej metody w jedną poprzez **złączenie** - what do you think about that?